web-app/src/app/tabs/controls/controls.component.html

```diff
@@ -1,3 +1,122 @@
+	    				</span>
+	    			</ng-template>
+				</span>
+				<span class="container-column-data spacing">
+					<span class="off data" [ngClass]="{'on' : controls.headlightsHigh == true, 'off' : controls.headlightsHigh == false}"><span class="dataUnderline">High</span></span>
+					<span *ngIf="lights.highBeams; else lightsHeadlightsNotHigh" class="lightsHeadlights">
+							<img src="../../../assets/HighHeadlightOn.png" alt="Headlight on" class="lightIcon">
+						</span>
+		    			<ng-template #lightsHeadlightsNotHigh>
+		    				<span><img src="../../../assets/HighHeadlightOff.png" alt="Headlight off" class="lightIcon"></span>
+		    			</ng-template>
+				</span>
+			</div>
+			<span class="spacing">
+				<div class="container-row-not-centered">
+					<span class="container-column-data">
+						<span class="off data" [ngClass]="{'on' : controls.signalLeft == true, 'off' : controls.signalLeft == false}"><span class="dataUnderline">Left </span></span>
+						<span *ngIf="lights.leftSignal; else lightsSignalNotLeft">
+							<img src="../../../assets/TurnSignalLeftOn.png" alt="Left signal on" class="lightIcon">
+						</span>
+	    				<ng-template #lightsSignalNotLeft>
+	    					<span><img src="../../../assets/TurnSignalLeftOff.png" alt="Left signal off" class="lightIcon"></span>
+	    				</ng-template>
+					</span>
+	    			<div class="signalSpacing"></div>
+	    			<span class="container-column">
+						<span class="off data" [ngClass]="{'on' : controls.signalRight == true, 'off' : controls.signalRight == false}"><span class="dataUnderline">Right</span></span>
+						<span *ngIf="lights.rightSignal; else lightsSignalNotRight">
+							<img src="../../../assets/TurnSignalRightOn.png" alt="Right signal on" class="lightIcon">
+						</span>
+	    				<ng-template #lightsSignalNotRight>
+	    					<span><img src="../../../assets/TurnSignalRightOff.png" alt="Right signal off" class="lightIcon"></span>
+	    				</ng-template>
+					</span>
+	    		</div>
+			</span>
+			<span class="spacing off" [ngClass]="{'on' : controls.hazard == true, 'off' : controls.hazard == false}"><span class="dataUnderline">On</span></span>
+		</div>
+		<div class="titleUnderline"></div>
+		<div class="container-row grey">
+			<span class="spacing dataHeader">Interior</span>
+			<span class="spacing dataHeader">Push To Talk</span>
+			<span class="spacing dataHeader">Horn</span>
+		</div>
+		<div class="container-row black">
+			<span class="spacing off data" [ngClass]="{'on' : controls.interior == true, 'off' : controls.interior == false}"><span class="dataUnderline">On</span></span>
+			<span class="spacing off data" [ngClass]="{'on' : controls.pushToTalk == true, 'off' : controls.pushToTalk == false}"><span class="dataUnderline">On</span></span>
+			<span class="spacing off data" [ngClass]="{'on' : controls.horn == true, 'off' : controls.horn == false}"><span class="dataUnderline">On</span></span>
+		</div>
+		<div class="container-row grey">
```

Comment:
Is there a `grey` stylesheet?

Reply:
There used to be, we switched to all black. I'll delete that